AntistasiOfficial.Altis/Functions/fn_getHighSupplies.sqf

```diff
@@ -0,0 +1,21 @@
+params ["_marker", "_threshold"];
+
+_city = [[ciudades, _marker] call BIS_fnc_nearestPosition, _marker] select (typeName _marker == typeName "");
+_data = server getVariable _city;
+if !(_data isEqualType []) exitWith {diag_log format ["Error in getHighSupplies. Passed %1 as reference.", _location]};
+
+_result = [];
+_supplyLevels = _data select 4;
+_tiers = ["CRITICAL", "LOW", "GOOD"];
+
+_indexThreshold = _tiers find _threshold;
```

Comment:
I've analyzed two cases:

    15:47:36 "_indexThreshold = 2"
    15:47:36 "_result = []"

and this

    15:47:37 "_indexThreshold = 1"
    15:47:37 "_result = [""CRITICAL"",""LOW"",""GOOD""]"

I see "" which i think it's  the mistake...

Reply:
don't understand, this function should not be directly related to resourcechecker